Automattic/wp-calypso
# CPT: Auto-select terms when added from the editor sidebar.

This branch seeks to auto-select newly added hierarchical terms in the editor sidebar after they have been persisted to the API.  Currently you can add a new term, but the new term is not checked in the `<EditorTermSelector />`.

__To Test__
- Open up a Portfolio in the editor
- Expand the 'Project Type' accordion, and click "Add New Project Type"
- Fill out the term form and save
- Confirm the new term is added and auto-selected

__Open Questions__
We still don't handle any issues encountered when saving fails on the API.  Is the state tree in the UI for this `terms/added` too much?

Test live: https://calypso.live/?branch=update/terms/auto-select-added-term
